api.planx.uk/modules/saveAndReturn/service/utils.ts

```diff
@@ -97,7 +97,16 @@ const validateSingleSessionRequest = async (
             team {
               name
               slug
-              settings: team_settings
+              settings: team_settings {
```

Comment:
Columns need to be specified for relationships. This response now matches the `TeamSettings` type.